docs/technical_reference/unit_models/electrolyzer.rst

```diff
@@ -0,0 +1,245 @@
+   "anode electrochemical potential :math:`^a`", ":math:`E_{ano}`", "anode_electrochem_potential", "None", ":math:`V`"
+   "anode overpotential", ":math:`\eta_{ano}`", "anode_overpotential", "None", ":math:`V`"
+   "anode stoichiometry of the reaction :math:`^{ab}`", ":math:`\varepsilon_{ano,j}`", "anode_stoich", "[p, j]", ":math:`\text{dimensionless}`"
+   "cathode area", ":math:`A_{cat}`", "cathode_area", "None", ":math:`m^2`"
+   "cathode current density", ":math:`i_{cat}`", "cathode_current_density", "None", ":math:`\frac{A}{m^2}`"
+   "cathode electrochemical potential :math:`^{ab}`", ":math:`E_{cat}`", "cathode_electrochem_potential", "None", ":math:`V`"
+   "cathode overpotential", ":math:`\eta_{cat}`", "cathode_overpotential", "None", ":math:`V`"
+   "cathode stoichiometry of the reaction :math:`^a`", ":math:`\varepsilon_{cat,j}`", "cathode_stoich", "[p, j]", ":math:`\text{dimensionless}`"
+   "current", ":math:`I`", "current", "None", ":math:`A`"
+   "cell voltage", ":math:`V_{cell}`", "voltage_cell", "None", ":math:`V`"
+   "ohmic resistance", ":math:`R`", "resistance", "None", ":math:`\Omega`"
+   "power", ":math:`P`", "power", "None", ":math:`W`"
+   "reversible voltage", ":math:`V_{rev}`", "voltage_reversible", "None", ":math:`V`"
+   "electrons contributing to electrolysis reactions", ":math:`\dot{n}_{e^-}`", "electron_flow", "None", ":math:`\frac{mol}{s}`"
+   "current efficiency", ":math:`\theta_{current}`", "efficiency_current", "None", ":math:`\text{dimensionless}`"
+   "voltage efficiency", ":math:`\theta_{voltage}`", "efficiency_voltage", "None", ":math:`\text{dimensionless}`"
+   "power efficiency", ":math:`\theta_{power}`", "efficiency_power", "None", ":math:`\text{dimensionless}`"
+   "molar flow of species j across the membrane from anolyte to catholyte", ":math:`\dot{n}_{mem,j}`", "mass_transfer_term", "[t, p, j]", ":math:`\frac{mol}{s}`"
+   "molar generation of species j by electrolysis at the anode", ":math:`\dot{n}_{ano,j}`", "custom_reaction_anode", "[t, j]", ":math:`\frac{mol}{s}`"
+   "molar generation of species j by electrolysis at the cathode", ":math:`\dot{n}_{cat,j}`", "custom_reaction_cathode", "[t, j]", ":math:`\frac{mol}{s}`"
+
+| :math:`^a` Variable intended to be move to a interchangeable component block, callable to the base electrolyzer model
+| :math:`^b` Value is normalized to 1 electron in electrolysis stoichiometry
+
+
+For non-fixed efficiencies, electrochemical potentials, and other relevant variables, custom constraints  may be constructed at the flowsheet level. These may allow for predictive performance as a function of temperature, concentration, overpotential, and other variables.
+
+.. _electrolyzer_equations:
+
+Equations
+-----------
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "membrane current density", ":math:`I = i_{mem}A_{mem}`"
+   "ion permeation through the membrane", ":math:`\dot{n}_{mem,j} = -t_{mem,j}\dot{n}_{e^-}`"
```

Comment:
I haven't read through the codes in detail.  But I think this equation overlaps the mass balance eq, where you've considered the flux across membrane as the mass_transfer term I assume.

Reply:
Overlaps isn't necessarily correct, but my reasoning is that, there technically is no variable constructed for a few variables at the unit model level. Rather, they are calculated by a variable within the control volume. I don't think we have a convention for these cases, so here I denote a _n_ as a flow rate term, but it matches the `mass_transfer_term`. However since `mass_transfer_term` is not exclusively membrane transfer, I think it may also be confusing to use some universal symbol as such.

I think this is a matter of is the documentation used to help users understand the model at the surface, or should it explain line by line and discuss every touched function, i.e., `ControlVolume0D` usage, or not. I'm going to make some changes based on your comments here once I figure out how to present it with more clarity. Open to suggestions.

Reply:
If the electron flow serving as a intermediate variable is what you meant, it's technically ok.  I am sure about this statement "mass_transfer_term is not exclusively membrane transfer"- so if your control vol is 3-D, membrane in 2-d is not in your c.v.. The mass_transfer_term of the c.v.  is exactly the membrane transfer because it is a 2-d flux in this case.

Reply:
I am realizing now that other unit models are also missing this, but I think it'd be beneficial to include the constrain name as a column for the table for users to more easily compare the documentation with model code (similar to how we provide variable_name in the variables table).

Reply:
@adam-a-a I would say maybe make it an issue. I don't want to change it here and be inconsistent with all other documentation. If this were to become a convention across unit models, maybe it would be best to have a helper function or harness to start these tables, and then we can do it for all unit models instead of this one.

Reply:
@hunterbarber sounds completely reasonable. I had some sort of helper function to do this for all our zero-order models, and there is a module in IDAES that could be used to display expressions in latex, if we wanted to go a step further.